ui/mark_ii_idle.qml

```diff
@@ -75,16 +75,29 @@ Mycroft.Delegate {
             width: gridUnit * 3
         }
 
-        // The date of the most recent Mark II build.  Only displayed for developers.
+        // The date and time of the most recent Mark II build.  Only displayed for developers.
         HomeScreenLabel {
-            id: homeScreenBuildDate
+            id: homeScreenBuildDateTime
             anchors.bottom: parent.bottom
             anchors.left: parent.left
             anchors.leftMargin: gridUnit * 5
             fontSize: 22
             fontStyle: "Regular"
             heightUnits: 3
-            text: sessionData.buildDate
+            text: sessionData.buildDateTime ? "Build: " + sessionData.buildDateTime : ""
```

Comment:
We'll need to update the scalable qml with the new variable name also

Reply:
yup.  my bad.